packages/backend/src/graphql/schema.graphql

```diff
@@ -1071,8 +1071,6 @@ input CreateQuoteInput {
   receiver: String!
   "Unique key to ensure duplicate or retried requests are processed only once. See [idempotence](https://en.wikipedia.org/wiki/Idempotence)"
   idempotencyKey: String
-  "ID of the tenant"
```

Comment:
`tenantId` got removed from the quote resolver input for backwards compatibility issues and it can be retrieved from the wallet address anyways

Reply:
> it can be retrieved from the wallet address anyways

I think perhaps this is the better pattern. It's simpler and I don't think we need to verify against the requestor's expectation for what the `tenantId` should be. 

I remember you mentioning that before and not having much of an opinion but I came across it myself when dealing with checking access for the createQuote resolver and felt like passing it in wasn't really providing any value, and it does make the gql input less complicated (IE requiring it only for operators).